### ISSUE TITLE
[All] update node-role.kubernetes.io/master=>node-role.kubernetes.io/control-plane

### ISSUE THREAD

**Issue by jichenjc** [kubernetes/cloud-provider-openstack#1839]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
> /kind bug
> /kind feature

**What happened**:

https://github.com/kubernetes/kubeadm/issues/2200
https://github.com/kubernetes-sigs/cluster-api/pull/5919
has further info

we have following occurrance need to be updated 

```
# grep node-role.kubernetes.io * -r
charts/openstack-cloud-controller-manager/values.yaml:#   node-role.kubernetes.io/controlplane: "true"
charts/openstack-cloud-controller-manager/values.yaml:#   - key: node-role.kubernetes.io/controlplane
charts/openstack-cloud-controller-manager/values.yaml:#   - key: node-role.kubernetes.io/etcd
docs/magnum-auto-healer/using-magnum-auto-healer.md:        node-role.kubernetes.io/master: ""
manifests/controller-manager/openstack-cloud-controller-manager-ds.yaml:        node-role.kubernetes.io/master: ""
manifests/controller-manager/openstack-cloud-controller-manager-ds.yaml:      - key: node-role.kubernetes.io/master
manifests/magnum-auto-healer/magnum-auto-healer.yaml:        node-role.kubernetes.io/master: ""
pkg/ingress/controller/controller.go:   DeprecatedLabelNodeRoleMaster = "node-role.kubernetes.io/master"
pkg/autohealing/controller/controller.go:       LabelNodeRoleMaster = "node-role.kubernetes.io/master"
tests/playbooks/roles/install-cpo-occm/tasks/main.yaml:            - key: node-role.kubernetes.io/master

```

**What you expected to happen**:

**How to reproduce it**:

**Anything else we need to know?**:

**Environment**:
- openstack-cloud-controller-manager(or other related binary) version:
- OpenStack version:
- Others:

**Comment by jichenjc** [comment]
#1840 is for OCCM only, there are more need to be updated

**Comment by chrigl** [comment]
/close

fixed by #1875
